### PR TITLE
Rename simtrack to simvue

### PR DESCRIPTION
**Pull request by alahiff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ The exact contents of both of the above options can be obtained directly by clic
...
 ## Usage example
 ```
-from simtrack import Simtrack
+from simvue import Simvue
 
 ...
 
 # Using a context manager means that the status will be set to completed automatically
-with Simtrack() as run:
+with Simview() as run:
```
Comment (AbyAbraham21) [inline README.md:26]: Should be Simvue?

Comment (alahiff) [reply README.md:26]: Yes. For some reason I kept on writing Simview not Simvue :-)

**Review by AbyAbraham21 — COMMENTED** [review]
Spotted one type Simview should be Simvue. Also I think Handler class is very generic, I feel SimvueHandler would be better.

File: README.md (hunk 1 above)
Comment (alahiff) [reply README.md]: Fixed now

File: examples/logging-to-simvue.py [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-run = Simtrack()
+run = Simvue()
 
 run.init(tags=['logging'],
          description='Logging test')
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
-sth = SimtrackHandler(run)
+sth = Handler(run)
```
[the remark this answers is not in the document]
Comment (alahiff) [reply examples/logging-to-simvue.py:11]: I started thinking Handler seemed better than SimvueHandler, but yes, probably we should use SimvueHandler.

**Review by AbyAbraham21 — APPROVED** [review]

**Comment by alahiff** [comment]
Handler has now been changed to SimvueHandler.

**Pull request merged**